lib/src/components/camera/service.dart

```diff
@@ -32,14 +32,14 @@ class CameraService extends CameraServiceBase {
   Future<GetImageResponse> getImage(ServiceCall call, GetImageRequest request) async {
     final camera = _fromManager(request.name);
     final image = await camera.image(mimeType: MimeType.fromString(request.mimeType));
-    return GetImageResponse(mimeType: image.mimeType.toString(), image: image.raw);
+    return GetImageResponse(mimeType: image.mimeType.name, image: image.raw);
   }
 
   @override
   Future<GetPointCloudResponse> getPointCloud(ServiceCall call, GetPointCloudRequest request) async {
     final camera = _fromManager(request.name);
     final image = await camera.pointCloud();
-    return GetPointCloudResponse(mimeType: image.mimeType.toString(), pointCloud: image.raw);
+    return GetPointCloudResponse(mimeType: image.mimeType.name, pointCloud: image.raw);
```

Comment:
this was a bug i introduced while writing this service, now that we have tests I know for sure this needs to be changed. `name` is the value i want, `toString()` returned `"instance of \MimeType\"`.

Reply:
[feat] maybe override the `toString` method on MimeType to return the name?

Reply:
I considered that, i think grabbing `name` is fine for what we are expecting. overriding toString _might_ be overkill for this, open to it though. It would be more appropriate if we wanted to combine multiple fields from the type in a certain way IMO.

Reply:
I actually think it's super useful to override the `toString`, as it would allow people to debug log/print the mimetypes easier if they're making their own camera implementation. `print("Received mimetype: $mimetype")` vs `print("Received mimetype: ${mimetype.name}") which tbf isn't a lot, but feels more ergonomic IMO